app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class CommandNameError(Exception):
+    """CommandNameError"""
+
+
+class DoubleNameError(Exception):
+    """DoubleNameError"""
+
+
+class LanCommandError(Exception):
+    """LaneCommandError"""
+
+
+def copy_file(command: str) -> None:
+    command_elements = command.split()
+
+    if len(command_elements) < 3:
+        raise LanCommandError(
+            "it must start with 'cp' and include two file names"
+            " 'cp input_file_name.txt output_file_name.txt'"
+        )
+
+    if command_elements[0] != "cp":
+        raise CommandNameError(f"'{command_elements[0]}' is not 'cp' command")
+
+    if command_elements[1] == command_elements[2]:
+        raise DoubleNameError(
+            f"intput: {command_elements[1]} output: {command_elements[1]}"
+            f" must not have the same names"
+        )
+
+    with (
+        open(command_elements[1], "r") as file_input,
```

Comment:
Code Style Issue: It's better to use more descriptive variable names. `file_input` is a great example of a descriptive variable name. However, `file_output` could be more descriptive. Consider renaming it to `output_file` or `destination_file`.